src/functions-reference/real-valued_basic_functions.Rmd

```diff
@@ -1472,6 +1472,12 @@ of x and the natural exponentiation of y. \[
 `R` **`log_inv_logit`**`(T x)`<br>\newline
 natural logarithm of the inverse logit function of x
 
+<!-- R; log_inv_logit_diff; (T1 x, T2 y); -->
+\index{{\tt \bfseries log\_inv\_logit\_diff }!{\tt (T1 x, T2 y): R}|hyperpage}
+
+`R` **`log_inv_logit_diff`**`(T1 x, T2 y)`<br>\newline
+natural logarithm of the difference of the inverse logit function of x and the inverse logit function of y
```

Comment:
Ooof, I think we should have named this `log_diff_inv_logit` (log of difference of inverse logits) but I guess this is why the docs are needed

Reply:
Ah yeah.